magicblock-ledger/src/store/api.rs

```diff
@@ -1373,6 +1278,39 @@ impl Ledger {
     }
 }
 
+pub trait HasColumn<C>
```

Comment:
nit: Better name would be TruncateableColumn

Reply:
Disagree, this isn't only for columns on which we delete, this is implemented for every column we have - just return a corresponding &LedgerColumn.

In some cases we want to map passed generic `C` to its column, but without this trait there's no good way to do it. Maybe only matching C::Name to expected set of names, but this will result in function returning Option<LedgerColumn<C>> which is ugly imo.